### ISSUE TITLE
problem with subdomain (different apps)

### ISSUE THREAD

**Issue by gsmeira** [cpriego/valet-linux#219]
I'm trying to link **my-project.test** and **wiki.my-project.test** but I'm being redirected to **my-project.test** every time I access **wiki.my-project.test**. There is something I could do to solve this?

**Comment by jmarcher** [comment]
How are you doing this? With a symlink ?

**Comment by gsmeira** [comment]
Hi @jmarcher, I just accessed the directory of each app and runned the command `valet link [my-project]`. I started using Valet yesterday, so I don't know if is something I'm doing wrong or if this is the expected behavior. All my projects are working good, but I got this problem with subdomain projects.

In the nginx folder (that I found in `~/.valet/Nginx` I open the **project.test** configuration file and I found ***.my-project.test** so I think the **wiki.my-project.test** is being catched by this, so I tried to remove the ***.my-project.test** and restart valet, but doesn't work.

For testing, I created a subdomain name that starts with a letter that comes first than the **my-project.test**, and, in that case, the subdomain and the main domain worked just fine.

Any ideas?
